CODEOWNERS

```diff
@@ -5,4 +5,7 @@
 *benchmark* @haresh-suresh  @nkraemer-sysdig 
 
 # monitor
-*monitor*alert* @arturodilecce
\ No newline at end of file
+*monitor*alert* @arturodilecce
+
+# policies
+*secure*policy* @team-secure-detection
```

Comment:
Please review the codeowners, it looks like this team doesn't exist in the `sysdiglabs` organization.
Once this is solved, we can merge it.

Reply:
team-secure-detection was added to Sysdiglabs by Ruben Eguiluz

Reply:
I made the code owners file valid, please approve :)